ui/app/mainui/AppMain.qml

```diff
@@ -990,8 +990,12 @@ Item {
             id: activityCenterPopupComponent
             ActivityCenterPopup {
                 id: activityCenter
-                height: appView.height - 56 * 2 // TODO get screen size // Taken from old code top bar height was fixed there to 56
-                y: 56
+                // TODO get screen size // Taken from old code top bar height was fixed there to 56
+                property int _buttonSize: 56
```

Comment:
How do you know it will be 56?

Reply:
There are todo for what. We have no such constant in style/theme currently

Reply:
@caybro any other suggestions here? Cause otherwise we can go with this for now.

Reply:
Nope